pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -1036,11 +1036,11 @@ def cmd_class_associators(context, classname, options):
         raise pywbem_error_exception(er)
 
 
-def get_namespaces(context, namespaces, default_rtn=False):
+def get_namespaces(context, namespaces, default_all_ns=False):
     """
-    Returns either the namespaces defined in namespaces parameter or if
+    Returns either the namespaces defined in --namespaces parameter or if
     namespaces is empty, either all of the namespaces available in the server or
-    the value None depending on the value of the parameter default.
+    the value None depending on the value of the parameter default_rtn.
```

Comment:
Change `default_rtn` to `default_all_ns` ?

Reply:
DONE